Gemfile.lock

```diff
@@ -1,113 +1,108 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    active-fedora (6.7.3)
-      activesupport (>= 3.0.0)
+    active-fedora (6.8.0)
+      activesupport (>= 3)
       deprecation
       mediashelf-loggable
       nom-xml (>= 0.5.1)
-      om (~> 3.0.0)
-      rdf
-      rdf-rdfxml (= 1.0.1)
+      om (~> 3.0)
+      rdf (~> 1.1, >= 1.1.1.1)
+      rdf-rdfxml (~> 1.1)
       rsolr
       rubydora (~> 1.6, >= 1.6.5)
-    activemodel (4.2.7.1)
-      activesupport (= 4.2.7.1)
-      builder (~> 3.1)
-    activeresource (4.0.0)
-      activemodel (~> 4.0)
-      activesupport (~> 4.0)
-      rails-observers (~> 0.1.1)
-    activesupport (4.2.7.1)
+    activemodel (5.0.0.1)
+      activesupport (= 5.0.0.1)
+    activesupport (5.0.0.1)
+      concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (~> 0.7)
-      json (~> 1.7, >= 1.7.7)
       minitest (~> 5.1)
-      thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
-    assembly-utils (1.4.6)
-      activeresource
-      activesupport
-      csv-mapper
-      dor-services (~> 5.3)
-      dor-workflow-service (>= 1.3.1)
+    airbrussh (1.1.1)
+      sshkit (>= 1.6.1, != 1.7.0)
+    assembly-utils (1.5.0)
+      dor-services (~> 5.5)
       druid-tools (>= 0.2.6)
-      fastercsv
       nokogiri
     bagit (0.3.2)
       docopt (~> 0.5.0)
       validatable (~> 1.6)
-    blue-daemons (1.1.11)
-    bluepill (0.1.1)
-      activesupport (>= 3.2, < 5)
-      blue-daemons (~> 1.1)
+    bluepill (0.0.64)
```

Comment:
for some reason, bluepill got downgraded to 0.0.64. the current version is 0.1.2.